helm/cert-manager/charts/cert-manager-giantswarm-clusterissuer/templates/_helpers.tpl

```diff
@@ -71,6 +71,11 @@ spec:
           environment: {{ .Values.acme.dns01.azureDNS.environment }}
           managedIdentity:
             clientID: {{ .Values.acme.dns01.azureDNS.identityClientID }}
+          {{- if not (empty .Values.acme.dns01.azureDNS.clientSecret) }}
```

Comment:
Why not just use `if` instead of `if not empty`?

Reply:
good idea 😃